File: JS/Car.js
class Car {
    id;
    license;
    driver;
    passenger;

    constructor ( license, driver ) {
        this.id;
        this.license = license;
        this.driver = driver;
        this.passenger;
    }

    printDataCar = () => {
        console.log(this.driver);
        // debugger
        console.log(this.driver.name);
        console.log(this.driver.document);
        console.log(this.license);
    }
}

File: Java/Car.java
class Car {
    
    private Integer id;
    private String license;
    private Account driver;
    private Integer passenger;

    public Car( String license, Account driver) {
        this.license = license;
        this.driver = driver;
        // passenger = 4;
    }

    void printDataCar() {
        if ( passenger !=  null ) {
            System.out.println(
                "License: " + license + 
                "Name Driver:" + driver.name +
                "Passengers:" + passenger
            );
        }
    }

    public Integer getPassenger() {
        return passenger;
    }

    public void setPassenger( Integer passenger ) {
        if ( passenger == 4) {
            this.passenger = passenger;
        } else {
            System.out.println("Necesitas asignar 4 pasajeros");
        }
    }

    
}